.circleci/config.yml

```diff
@@ -314,7 +314,7 @@ jobs:
           # install without building, then remove the tests and build, so that carthage
           # doesn't try to build the other integration tests
           command: |
-              ./carthage.sh update --no-build
+              ./carthage.sh bootstrap --no-build
```

Comment:
Is this change intended? It used to be `update` https://github.com/RevenueCat/purchases-ios/commit/caf58d14b715bf0b510f93e5fd7cae6dab912784#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R317

Reply:
Yup, initially I thought this was better, and it Worked On My Machine™️